news/2018-12-31-beatmap-spotlights-fall-2018.md

```diff
@@ -0,0 +1,250 @@
+﻿---
+layout: post
+title: "Seasonal Spotlights: Fall 2018"
+date: 2018-12-31 06:00:00 +0000
+---
+
+![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/header.jpg)
+
+The beatmap charts for the Fall Spotlights can be found [here](https://osu.ppy.sh/rankings/osu/charts?spotlight=252). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/beatmaps/packs?type=chart) for easier access. **The Fall 2018 Spotlights rankings will end on the 15th of March, 2019.**
```

Comment:
Intro paragraph is first, then the header.

Reply:
No intro in this version yet

Reply:
Did not read the content of this line, sorry.

Reply:
intro added with the latest commit